Files/App.xaml

```diff
@@ -138,7 +138,10 @@
                         </LinearGradientBrush.GradientStops>
                     </LinearGradientBrush>
 
-                    <!--  WinUI style is breaking us, overriding with a modified version solves crashes in release  -->
+					<x:Double x:Key="SidebarIconSize">20</x:Double>
+					<Thickness x:Key="NavigationViewContentPresenterMargin">0,0,0,0</Thickness>
```

Comment:
Can you add a comment that this is temporary?

Reply:
@yaichenbaum This likely isn't temporary because they've added that resource with default values. Either way, it doesn't matter as this will always be our desired value.